allennlp/data/dataset_readers/conll2003.py

```diff
@@ -61,24 +65,23 @@ class Conll2003DatasetReader(DatasetReader):
         Each will have its own namespace: ``pos_tags``, ``chunk_tags``, ``ner_tags``.
         If you want to use one of the tags as a `feature` in your model, it should be
         specified here.
-    coding_scheme: ``str``, optional (default=``IOB1``)
+    convert_to_coding_scheme: ``str``, optional (default=``None``)
         Specifies the coding scheme for ``ner_labels`` and ``chunk_labels``.
-        Valid options are ``IOB1`` and ``BIOUL``.  The ``IOB1`` default maintains
-        the original IOB1 scheme in the CoNLL 2003 NER data.
-        In the IOB1 scheme, I is a token inside a span, O is a token outside
-        a span and B is the beginning of span immediately following another
-        span of the same type.
+        A valid option is ``BIOUL``.
+        The default ``None`` maintains the original IOB1 scheme in the CoNLL 2003 NER data.
+        The ``BIOUL`` option recodes the data into the BIOUL scheme.
     label_namespace: ``str``, optional (default=``labels``)
         Specifies the namespace for the chosen ``tag_label``.
     """
     _VALID_LABELS = {'ner', 'pos', 'chunk'}
+    _VALID_CODING_SCHEMES = {'BIOUL',}
 
     def __init__(self,
                  token_indexers: Dict[str, TokenIndexer] = None,
                  tag_label: str = "ner",
                  feature_labels: Sequence[str] = (),
                  lazy: bool = False,
-                 coding_scheme: str = "IOB1",
+                 convert_to_coding_scheme: str = None,
```

Comment:
I think the type annotation on this might be `Optional[str]`? will need to import `Optional` from `typing`.

Reply:
Have we made any official decision on this?  Personally, I prefer leaving off the `Optional`, because that's already implied by the fact that the default value is `None`.  Is there a good reason to add in `Optional`?  @joelgrus, does it impact any `FromParams` or configurator stuff?